### PR TITLE
add log

### PR DESCRIPTION
**Pull request by coolbaobei**

### CODE REVIEW

File: apollo_test.go [hunk 1]
```diff
@@ -3,12 +3,11 @@
 package apollo
 
 import (
+	"apollo/internal/mockserver"
```
Comment (liweiv) [inline apollo_test.go:6]: 与标准库区分 隔开  单独起一行

**Review by liweiv — COMMENTED** [review]
add gopkg.in/logger.v1 to go mod list.

File: apollo.go [hunk 2]
```diff
@@ -3,6 +3,8 @@
 //Package apollo ctrip apollo go client
 package apollo
 
+import log "gopkg.in/logger.v1"
```
Comment (liweiv) [inline apollo.go:6]: add gopkg.in/logger.v1 to go mod list

Comment (coolbaobei) [reply apollo.go:6]: ok

**Pull request merged**